pkg/thing/interactors/list_things_test.go

```diff
@@ -62,10 +62,10 @@ var ltCases = []listThingsTestCase{
 	{
 		"failed to publish list things response",
 		"authorization key",
-		errors.New("Message queue unavailable"),
+		errors.New("message queue unavailable"),
 		[]*entities.Thing{},
 		nil,
-		errors.New("Message queue unavailable"),
```

Comment:
Add in the commit message that you're fixing this too.

Reply:
I putted a message body to describe this :smile: